mixed_pond_experimental/bechet_etal.py

```diff
@@ -0,0 +1,257 @@
+Volume = 6153.05 #m3
+area = 4047 #m2
+
+T_wc_0 = 19.3
+
+
+#Tp: pond temp (K)
+#Pw: density (kh m-3)
+#Cpw: specific heat capacity (J kg-1 K-1)
+#V: pond volume (m3)
+#Qra,p: radiation from pond surface (W)
+#Qra,s: total (direct and diffuse) solar radiation (W)
+#Qra,a: radiation from air to pond (W)
+#Qev: evaporative heat flux (W)
+#Qconv: convective flux at pond surface (W)
+#Qcond: conductive flux with ground at pond bottom (W) #not using
+#Qi: heat flux associated with water inflow (W)
+#Qr: heat flux induced by rain (W) #not using
+
+
+#open csv file using pandas to create pandas dataframe 
+data = pd.read_csv("bechet_etal_input.csv") 
+
+def find_day_month_year(input_day, start_day = dt.date(2016,12,31)):
+    computed_day = start_day + dt.timedelta(days = float(input_day))
+    days_from_year_start = computed_day - dt.date(computed_day.year, 1, 1) + dt.timedelta(days = 1)
+
+    return days_from_year_start.days, computed_day
+
+#create a function to read data for particular day, month and year so we can use it to loop through all days later
+
+def read_dataline(day_argue, hour):
+    
+    day, day_mon_year = find_day_month_year(day_argue)
+    #year = day_mon_year.year
+    
+    selected_data = data[(data['DAY']== day) & (data['HR'] == hour)]
+        
+    return selected_data 
+
+#pond radiation
+
+def calculate_Qrap(T_wk):
+    
+    #Qra,p = -εwσTp4S 
+    ew = 0.97 #water emissivity
+
+    roh1 = 5.67 * (10**(-8)) #Wm-2K-4 or Joule/second*m2K4
+    
+    #roh = roh1 #joule/hr*m2*K4
+    SA = 4047 #m2 area
+    Qrap = -ew * roh1 * SA * (T_wk**4)
+    
+    return Qrap
+
+#solar radiation
+#Qra,s = (1-fa)HsS
+
+def calculate_Qras(day_argue, hour):
+    daily_data = read_dataline(day_argue, hour)
+    solrad = float(daily_data['SRAD'])
+    
+    fa = 2.5 #% algae absorption
+
+    Hs =  solrad #* 3600 #solar radiation W/m2 or joule/second*m2
+    
+    SA = 4047 #m2 area
+    Qras = (1-fa) * Hs* SA
+    
+    return Qras
+
+#air radiation
+#Qra,a = εw εa σTa4S
+
+def calculate_Qraa(day_argue, hour):
+    daily_data = read_dataline(day_argue, hour)
+    T_a = float(daily_data['T2M'])+ 273.15 #check this #kelvin
+    
+    ew = 0.97 #% algae absorption
+    
+    ea = 0.8 #air emissivity
+    
+    roh1 = 5.67 * (10**(-8)) #Wm-2K-4 or Joule/second*m2K4
+    
+    #roh = roh1 * 3600 #joule/day*m2*K4
+
+    
+    SA = 4047 #m2 area
+    Qraa = ew * ea* roh1* (T_a**4) * SA
+    
+    return Qraa
+
+#evaporation
+#Qevap = -meLwS 
+
+def calculate_Qevap(day_argue, hour):
+    daily_data = read_dataline(day_argue, hour)
+    T_a = float(daily_data['T2M']) +273.15#kelvin
+    RH = float(daily_data['RH2M'])
+    v = float(daily_data['WS2M']) #*3600 #m/s
+    
+    Lw = 2.45 * (10**6) #water latent heat J/kg 
+    #this is site specific check with Drew
+    Dw = 2.4 * (10**-5) #*3600 #m/s 
+    l = 63.61#pond length in m
+    va = 1.5 *(10**-5) #*3600  #m2/s
+    
+    Rel = (l*v)/va
+    SH= (math.sqrt(0.035 * (Rel**0.8)))**3
```

Comment:
I understand the eqn for SH to be

SH = 0.035 * (Rel**0.8) * (Sch**(1/3))
where Sch = va/Dw

Unclear where you got the above equation?